File: backend/models/Guide.js
const mongoose = require("mongoose");

const GuideSchema = new mongoose.Schema({
  name: {
    type: String,
    required: true,
  },
  category: {
    type: String,
    required: true,
  },
  summary: {
    type: String,
    required: true,
  },
  image_url: {
    type: String,
    required: true,
  },
  file_url: {
    type: String,
    required: true,
  },
});

module.exports = mongoose.model("Guide", GuideSchema);

File: frontend/app/src/main/java/com/example/retech/databaseModel/Guide.kt
package com.example.retech.databaseModel

import androidx.room.Entity
import androidx.room.PrimaryKey
import java.io.Serializable

@Entity(tableName = "guides")
data class Guide(
    @PrimaryKey(autoGenerate = true)
    val id: Int = 0,
    val name: String,
    val category: String,
    val summary: String,
    val image_url: String,
    val file_url: String
) : Serializable



